### ISSUE TITLE
Scale issue in map plots

### ISSUE THREAD

**Issue by AugustinVintzileos** [esowc/ecPoint-Calibrate#135]
When plotting the mean forecast error the color scale is starts from -1C and goes up to 1000C (see figure). It would be more informative to have a scale e.g., between -10C to 10C with more colors.
![Fig](https://user-images.githubusercontent.com/72265581/101375513-7af6e380-387d-11eb-9060-d03cf40a822c.png)

**Comment by ATimHewson** [comment]
Good point Augustin - I completely agree - and maybe even wider than +/-10C for extreme winter-time situations. The green-black-yellow-red scale was designed for FER, to mirror the colours and values used on the mapping function histogram. But you are right now we need alternative settings for 2m temperature.
And on your top plot these settings were also found useful for what we did with rainfall, but maybe now with 2m temperature we need an adjustment, maybe incorporating some sort of logarithmic-style variation for higher frequencies.
Need to think carefully about how to adapt to different variables, and how many legend settings we would ideally like. Maybe introducing a set which we can select from (or manually add to later without any intervention from Anirudha) would be the way to go?

**Comment by AugustinVintzileos** [comment]
Hi Tim,

I think your idea of setting scales manually is the best. Having a predetermined set would facilitate the task but we certainly need the ability to change as necessary.

Best,

Augustin

________________________________
